cfg/CFG.cc

```diff
@@ -135,7 +136,7 @@ void CFG::sanityCheck(core::Context ctx) {
         ENFORCE(thenCount == 1, "bb id={}; then has {} back edges", bb->id, thenCount);
         ENFORCE(elseCount == 1, "bb id={}; else has {} back edges", bb->id, elseCount);
         if (bb->bexit.thenb == bb->bexit.elseb) {
-            ENFORCE(!bb->bexit.cond.variable.exists());
+            ENFORCE(bb->bexit.cond.variable == core::LocalVariable::unconditional());
```

Comment:
This is inside `CFG::sanityCheck`. This means that we ENFORCE that `bb->bexit.thenb == bb->bexit.elseb` implies `variable == <unconditional>` and vice versa.